x-pack/test/observability_ai_assistant_api_integration/tests/chat/chat.spec.ts

```diff
@@ -99,9 +99,11 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             });
 
             for (let i = 0; i < NUM_RESPONSES; i++) {
-              await simulator.next(`Part: i\n`);
+              await simulator.next(`Part: ${i}\n`);
             }
 
+            await simulator.tokenCount({ completion: 20, prompt: 33, total: 53 });
```

Comment:
Did you manually calculate these? If the input chunks are changed, the token counts will go out of sync without us noticing, right? Theres nothing that actually counts the tokens and require the token count to match?